data/notices.json

```diff
@@ -1,5 +1,21 @@
 {
   "notices": [
+    {
+      "title": "aws-iam: IAM OIDC provider configures the wrong certificate thumbprint",
+      "issueNumber": 8607,
+      "overview": "Your application is using the iam.OpenIDConnectProvider resource, which currently configures the thumbprint of the leaf certificate instead of the root. Since leaf certificates are rotated more frequently, you are in danger of application errors while trying to use the provider to authenticate against AWS services. See the attached issue for mitigation steps and apply them as soon as possible.",
```

Comment:
```suggestion
      "overview": "Your application is using the iam.OpenIDConnectProvider resource, which currently configures the thumbprint of the leaf certificate instead of the root. Since leaf certificates are rotated more frequently, you are in danger of application errors next time those certificates get rotated. See the attached issue for mitigation steps and apply them as soon as possible.",
```

?

Reply:
Looks good to me. Will update.